cve_bin_tool/outputs/CSVOutput.py

```diff
@@ -0,0 +1,71 @@
+import csv
+import os
+
+from cve_bin_tool.outputs.Output import Output
+
+
+class CSVOutput(Output):
+    def __init__(self, filename, modules, logger):
+        super().__init__(filename, "csv")
+        self.logger = logger
+        self.modules = modules
+
+    def create_output(self):
+        return self.output_csv()
+
+    def output_cves(self, outfile):
+
+        """ Output a list of CVEs
+        format is modules[checker_name][version] = dict{id: severity}
+        """
+        # if the output is csv we must open a file
+        outfile = open(f"{outfile}", "w")
+
+        writer = csv.writer(outfile)
+        writer.writerow(["Module Name", "Version", "CVE Number", "Severity"])
+
+        writer = csv.writer(outfile)
+        for modulename, versions in self.modules.items():
+            for version, cve_list in versions.items():
+                for cve_number, cve_severity in cve_list.items():
+                    row = [modulename, version, cve_number, cve_severity]
+                    writer.writerow(row)
+
+        # We must also close the file
+        outfile.close()
```

Comment:
@PrajwalM2212 this function was actually used to output the list of cves to the console. The output_csv function was almost same that was the only reason output_csv called this function. As now you have sperated all the files. It's better to merge the functionality of this function to the output_csv function.

Reply:
@SinghHrmn merging this function to `output_csv` will lead to one large function.  I think we can keep them as two functions for readability/maintenance purposes.